airbyte-integrations/java-template-destination/readme.md

```diff
@@ -0,0 +1,20 @@
+### How to write a destination integration in java.
```

Comment:
This is a good start. Some things I would love for us to have soon to have the best DX possible (though not in this PR): 
1. A description of how airbyte core interacts with integrations & the contract of an integration image (`write/spec/discover/check` etc.)
1.  Having a script that automates this mechanical setup

Will create an issue for this.

Reply:
cool! i have create an issue for the writing the script: https://github.com/airbytehq/airbyte/issues/512